src/webgl/p5.RendererGL.Retained.js

```diff
@@ -3,6 +3,38 @@
 'use strict';
 
 var p5 = require('../core/main');
+require('./p5.RendererGL');
+
+// a render buffer definition
+function BufferDef(size, src, dst, attr, map) {
+  this.size = size; // the number of FLOATs in each vertex
+  this.src = src; // the name of the model's source array
+  this.dst = dst; // the name of the geometry's buffer
+  this.attr = attr; // the name of the vertex attribute
+  this.map = map; // optional, a transformation function to apply to src
+}
+
+var _flatten = p5.RendererGL.prototype._flatten;
+var _vToNArray = p5.RendererGL.prototype._vToNArray;
+
+var strokeBuffers = [
+  new BufferDef(3, 'lineVertices', 'lineVertexBuffer', 'aPosition', _flatten),
+  new BufferDef(4, 'lineNormals', 'lineNormalBuffer', 'aDirection', _flatten)
+];
+
+var fillBuffers = [
```

Comment:
attach stroke and fill buffers to `RendererGL` as well for consistency?

Reply:
The only reason the text buffers are exposed here is because they're needed by the text rendering code which is in a separate file. Ideally even that won't be necessary in the future, but for now I'd rather err on the side of keeping more stuff hidden if possible.